src/components/NumberChart.jsx

```diff
@@ -77,22 +77,22 @@ export default class NumberCharts extends React.Component {
         const lowValueColor = config.lowValueColor || currentTheme.number.style.labels.lowValue.fill;
 
         return (
-            <svg height="100%" width="100%" viewBox={`0 0 ${width} ${height}`}>
+            <svg height="100%" width="100%" viewBox={`0 0 800 450`}>
                 <VictoryLabel
                     textAnchor="middle"
                     verticalAnchor="middle"
                     x="50%"
                     y="25%"
                     text={config.title}
-                    style={{ fill: currentTheme.number.style.labels.title.fill, fontSize: width / 20 }}
+                    style={{ fill: currentTheme.number.style.labels.title.fill, fontSize: 40 }}
```

Comment:
> When the user gives inappropriate values chart data overlaps, so we are going to auto scales to available outer wrapper width/height, preserving aspect ratio.

Then shouldn't the `fontSize` value be calculated based on `width` & height`? Or from the aspect ration?

Reply:
@erangatl Are we going to change this?

Reply:
This is the max font size we allow and this scales down with the title length to fit to available space